### DISCUSSION TITLE
kb/en/index

### DISCUSSION THREAD

**Discussion by giscus** [jiegec/kb#17] [General]
# kb/en/index

Knowledge Base of @jiegec

https://jia.je/kb/en/index.html

**Comment by MingHaiMr** [comment]
nice

**Comment by zssfred** [comment]
do you have an email or any social network?

**Reply by jiegec** [reply]
You can find it on https://github.com/jiegec

**Comment by bit-capybara** [comment]
awesome lastest data wiki. thank u!
